.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug Report
+about: Report a bug encountered while operating Kubernetes federation v2
+
+---
+
+<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!-->
+
+
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+**Anything else we need to know?**:
+
+**Environment**:
+- Kubernetes version (use `kubectl version`):
```

Comment:
I think environment-wise the list should be different from kubernetes.  How about the following list instead?

 - Kubernetes version (use `kubectl version`)
 - Federation version
 - Install method (helm vs script)
 - Install configuration (namespaced vs cluster-scoped)
 - Others